src/App.js

```diff
@@ -0,0 +1,113 @@
+import WaitingList from './WaitingList.js';
+import FinishedList from './FinishedList.js';
+import AddTodoBtn from './AddTodoBtn.js';
+
+function App({app, initialState, todoID}) {
+
+    this.todoID = localStorage.getItem('todoID') ? JSON.parse(localStorage.getItem('todoID')) : todoID;
+    this.state = {
+        waitingTodo: localStorage.getItem('waitingTodo') ? JSON.parse(localStorage.getItem('waitingTodo')) : initialState.waitingTodo,
+        finishedTodo: localStorage.getItem('finishedTodo') ? JSON.parse(localStorage.getItem('finishedTodo')) : initialState.finishedTodo,
+    };
+    this.setState = (nextState) => {
+
+        this.state = {
+            ...this.state,
+            ...nextState
+        }
+
+        localStorage.setItem('todoID', JSON.stringify(this.todoID));
+        localStorage.setItem('waitingTodo', JSON.stringify([...this.state.waitingTodo]));
+        localStorage.setItem('finishedTodo', JSON.stringify([...this.state.finishedTodo]));
+
+        this.render();
+    }
+
+    this.onAddTodo = (value) => {
+        this.todoID++;
+        this.setState({
+            waitingTodo: [
+                {
+                    todoID: this.todoID,
+                    todoContent: value,
+                },
+                ...this.state.waitingTodo,
+            ],
+        });
+    }
+
+    this.onClickWaitingTodoItem = ({id, todoContent}) => {
```

Comment:
```suggestion
    this.onWaitingTodoItemClick = ({id, todoContent}) => {
```

컨벤션이긴 하나 영어 어순상 자연스럽게 `on` + `TARGET` + `EVENT`로 보통 짓는 것 같습니다!
`onChangeText` ->  `onTextChange`